### PR TITLE
Removed Scopes validation in auth service

### PR DESCRIPTION
**Pull request by lesagi**

### CODE REVIEW

File: src/services/auth/index.ts [hunk 1]
```diff
@@ -24,9 +24,7 @@ export class Auth implements AuthDriver {
 	public async isAuthorized(request: FastifyRequest, scopes: Scope[]): Promise<boolean> {
 		return this._driver.isAuthorized(request, scopes)
 	}
-	public async allowedScopes(request: FastifyRequest): Promise<Scope[]> {
```
Comment (shay123g) [inline src/services/auth/index.ts:27]: @lesagi  should it be removed only from here?

**Review by shay123g — COMMENTED** [review]
this is the only file to be modified?

**Comment by SnirShechter** [comment]
@lesagi 

Following our talk, can you take a look at the [feature/auth-service](https://github.com/mini-services/miniurl/tree/feature/auth-service) branch? I think everything besides the configuration is already done there. 

You just need to add the configuration to `config/index.ts`, validate it in `config/validate.ts`, then pass it to the auth-service in `index.ts`.

Can you do it?

**Comment by SnirShechter** [comment]
#110 Already adds this.

**Pull request closed without merging**